website/docs/cloud-docs/workspaces/change-requests/manage.mdx

```diff
@@ -41,6 +41,8 @@ After creating your change request, HCP Terraform notifies you if it has success
 
 Open a workspace and select **Change requests** from the side navigation to display a list of that workspace's active change requests. You can view the full description of a change request by clicking on that request. If someone creates a change request on multiple workspaces, each workspace has its own independent copy of that change request.
 
+If unarchived change requests exist on a workspace, the change request navigation will have a badge indicating how many.
```

Comment:
this bit

Reply:
Small rephrase for flow, but otherwise great!
```suggestion
The change request navigation displays a badge indicating how many unarchived change requests exist on a workspace.
```